src/DotLiquid/Condition.cs

```diff
@@ -128,6 +144,36 @@ private static bool EqualVariables(object left, object right)
             return left.SafeTypeInsensitiveEqual(right);
         }
 
+        private static bool BlankVariable(object obj)
+        {
+            if (obj == self)
+                return false;
+            switch (obj)

```

Comment:
this can simply be placed in Context.Resolve method, e.g
```
                case "blank":
                    return new Symbol(o => {
                        switch (o)
                        {
                            case null:
                            case bool b when b == false:
                            case string s when string.IsNullOrWhiteSpace(s):
                            case IEnumerable e when !e.GetEnumerator().MoveNext():
                                return true;
                            default:
                                return false;
                        }
                    });
```